seeders/20200206005427-demo-patient.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('patients', [{
```

Comment:
make sure all fields from the schema are accounted for here - you can also create different patients that are missing nullable fields